docs-ref-conceptual/use-cli-effectively.md

```diff
@@ -250,6 +250,8 @@ Proxy is common behind corporate network or introduced by tracing tools like Fid
 
    A frequent ask is whether or not `HTTP_PROXY` or `HTTPS_PROXY` environment variables should be set, the answer is it depends. For Fiddler on Windows, by default it acts as system proxy on start, you don't need to set anything. If the option is off or using other tools which don't work as system proxy, you should set them. Since almost all traffic from CLI is SSL-based, only `HTTPS_PROXY` should be set. If you are not sure, just set them, but do remember to unset it after the proxy is shut down. For fiddler, the default value is `http://localhost:8888`.
 
+   Some proxies require authentication and therefore the format of the `HTTP_PROXY` or `HTTPS_PROXY` environment variables should include the authentication, e.g., `HTTPS_PROXY="https://username:password@proxy-server:port"`. This is needed by the underlying python libraries. For details, see [How to configure proxies for the Azure libraries](https://docs.microsoft.com/azure/developer/python/azure-sdk-configure-proxy?tabs=bash) 
```

Comment:
The link has now become https://learn.microsoft.com/en-us/azure/developer/python/sdk/azure-sdk-configure-proxy?tabs=bash

Actually, the content is wrong. 

![image](https://user-images.githubusercontent.com/4003950/233890204-ece556c8-373c-4036-aeda-c29226f25184.png)

Environment variables on Linux should be set with `export`.